### ISSUE TITLE
Drop-down menu in Message Index bottom buttons doesn't work

### ISSUE THREAD

**Issue by Hatshepsut71** [SychO9/smf-potato-theme#11]
When I click on right arrow of bottom buttons in Message Index, nothing happens, and drop-down menu does not appear.
Drop-down menu of top buttons works as well.
I will attach a screenshot from my test forum.
![2020-09-25 (1)](https://user-images.githubusercontent.com/68983397/94233096-0237ea80-ff10-11ea-923a-980f8bb357e8.png)

**Comment by SychO9** [comment]
Interesting, I cannot reproduce this in my local setup, could you please check the browser console for any errors when you try opening the dropdown ?

**Comment by Hatshepsut71** [comment]
Yes, I will attach a screenshot with errors in the browser console
![2020-09-25 (2)](https://user-images.githubusercontent.com/68983397/94264010-a20c6d00-ff3e-11ea-98ea-18cb0a86fac3.png)

**Comment by SychO9** [comment]
I took a look at your forum to see the error for myself, it seems the error in the console is due to a script in your page

```js
<script>
		jQuery(document).ready(function($) {
			if ($.browser.opera && $("div#tooltip")) // <-- triggered from here
				$("a#toTop, a#toBottom").removeAttr("title"); // temp. toggle for Opera
			let nav = $.browser.mozilla ? 1 : 0; // Fix for Gecko
...
```

Dunno which mod that comes from, but maybe try disabling it and see if the drop down works after that.

**Comment by Hatshepsut71** [comment]
In my test forum, I have only two mods installed - Tinyportal 2.0 and Scrolling Buttons 0.5.
I will unistall them one by one to see how the drop-down menu works.

**Comment by Hatshepsut71** [comment]
Yes, problem solved after uninstalling of Tinyportal 2.0

**Comment by SychO9** [comment]
ah, so it's an issue with Tinyportal, you should probably report it to them.
